uint16 RAM_N
uint16 RAM_D
uint16 CPU_usage_1
uint16 CPU_usage_2
uint16 CPU_usage_3
uint16 CPU_usage_4
uint16 CPU_usage_5
uint16 CPU_usage_6
float64 BCPU_temp
float64 MCPU_temp
float64 GPU_temp
float64 PLL_temp
float64 Tboard_temp
float64 Tdiode_temp
float64 PMIC_temp
int16 thermal
int16 VDD_IN_N
int16 VDD_IN_D
int16 VDD_CPU_N
int16 VDD_CPU_D
int16 VDD_GPU_N
int16 VDD_GPU_D
int16 VDD_SOC_N
int16 VDD_SOC_D
int16 VDD_WIFI_N
int16 VDD_WIFI_D
int16 VDD_DDR_N
int16 VDD_DDR_D
